### ISSUE TITLE
Proposal: Switch NAME in NAME-service to service name rather than package name

### ISSUE THREAD

**Issue by hasAdamr** [metaverse/truss#106]
Currently if you run truss on:
```
syntax = "proto3";
package Foo;

service Bar {
...
```

You will get a directory structure:
```
./Foo-service/Foo-server/
...etc...
```

It seems silly really, it was just one of the many decisions made during the code restructure of pull #49.

I propose that this is changed to the service name. In this case it would be:
```
./Bar-service/Bar-server/
...etc...
```

This change will cause `./truss/parsepkgname` to be modified to parse the service name rather than the package name. So while this change is pretty simple, it is not trivial.

**Comment by zaquestion** [comment]
:+1: DO IT

**Comment by hasIan** [comment]
This is a good idea. I just ran into this with proto files that have a package with a `pb` suffix, and the service directory ends up being called `myservicepb-service`. Yuck! This can be worked around with conscientious naming of packages, but only if stakeholders agree, so this is a good proposal no matter what.

**Comment by hasLeland** [comment]
Implemented in PR #120 
:+1: :100:
